packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Page/page';
+import { css } from '@patternfly/react-styles';
+import { BarsIcon } from '@patternfly/react-icons';
+import { Button, ButtonVariant } from '../../components/Button';
+
+export interface PageHeaderProps extends React.HTMLProps<HTMLDivElement> {
+  className?: string;
+  logo?: React.ReactNode;
+  logoProps?: object;
+  logoComponent?: React.ReactNode;
+  toolbar?: React.ReactNode;
+  avatar?: React.ReactNode;
+  topNav?: React.ReactNode;
+  showNavToggle?: boolean;
+  isNavOpen?: boolean;
+  onNavToggle?: () => void;
+  'aria-label'?: string;
+}
+
+export const PageHeader: React.FunctionComponent<PageHeaderProps> = ({
+  className='',
+  logo=null as React.ReactNode,
+  logoProps=null as object,
+  logoComponent = 'a',
+  toolbar=null as React.ReactNode,
+  avatar=null as React.ReactNode,
+  topNav=null as React.ReactNode,
+  isNavOpen=true,
+  showNavToggle=false,
+  onNavToggle=() => undefined,
```

Comment:
Change to `=> undefined as any`, not sure how it passed the typecheck!

Reply:
fixed